### PR TITLE
Address review comments for a prior PR

### PR DESCRIPTION
**Pull request by xornivore**
[Rendered](https://github.com/DataDog/dcos-datadog-agent-docs/blob/xornivore/address-review-comments-from-personal-repo/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,8 +200,8 @@ In order for the Datadog Agent Spark integration to be able to collect metrics i
 mode the Spark and Spark History Server packages have to be configured according to the D2IQ security recommendations
 for this mode of operation.
 
-The following instructions are provided as a reference only. If you have already installed Spark and History Server you
-may skip this section.
+> :warning: The following instructions are provided as a reference only. If you have already installed Spark and History
+> Server you may skip this section.
```
Comment (CharlyF) [inline README.md]: Right after you mention that one needs to confirm that HDFS is running with a service account, that there are permissions needed for the client and that there is a bare minimum list of permissions for spark. I don't know if one can install HDFS without all of these requirements, but if not, maybe not worth saying "skip the section". Could also add an anchor for people to skip to where you think they can.

File: deploy/dcos-node-datadog-agent.json [hunk 2]
```diff
@@ -1,102 +1,102 @@
...
+          "containerPort": 8125,
+          "hostPort": 8125,
+          "servicePort": 10000,
+          "protocol": "udp",
+          "labels": {}
         }
-    ],
-    "portDefinitions": [
+      ],
+      "privileged": false,
```
Comment (CharlyF) [inline deploy/dcos-node-datadog-agent.json:63]: Why don't we need to run as priviledged ? can we interact with the socket correctly ? (docker check working ok ?).
(I do like that we explicitely say `"mode": "RO"`).

File: deploy/dcos-node-datadog-agent.json [hunk 3]
```diff
@@ -1,102 +1,102 @@
...
+      },
+      "gracePeriodSeconds": 300,
+      "intervalSeconds": 60,
+      "timeoutSeconds": 20,
+      "maxConsecutiveFailures": 3
+    }
+  ],
+  "portDefinitions": [
+    {
+      "port": 10000,
```
Comment (CharlyF) [inline deploy/dcos-node-datadog-agent.json:91]: is that for a mesos service ? I see it referenced [here](https://github.com/DataDog/dcos-datadog-agent-docs/pull/1/files#diff-41c6a2fa5a4ebdf8033e4333b29006c05974cf2814d66e914b7f0631c1e8919dR58)

**Review by CharlyF — COMMENTED** [review]
One additional line we could edit:
l:247 in the README.md

> If the service account name is `spark` than Following would be the bare minimum list:

File: README.md (hunk 1 above)
Comment (xornivore) [reply README.md]: Great points, thank you 👍 HDFS and the client app are prerequisites for Spark History server so I should probably clarify that, will add a skip anchor as well.

File: deploy/dcos-node-datadog-agent.json (hunk 2 above)
Comment (xornivore) [reply deploy/dcos-node-datadog-agent.json:63]: Hm.. docker check does seem to be working, I don't think we'd need privileged for that?
```
    {
      "metric": "docker.uptime",
      "points": [
        [
          1602814475,
          174
        ]
      ],
      "tags": [
        "container_id:7d7c3a0ce2077259d4a60c6910bfccc4ba78af197cc15185dd4a8074ac643643",
        "container_name:mesos-668b5ec2-47ed-42d5-8aeb-fe937c425418",
        "docker_image:datadog/agent-dev:dcos-auth-token-py3",
        "image_name:datadog/agent-dev",
        "image_tag:dcos-auth-token-py3",
        "marathon_app:/datadog-agent",
        "mesos_task:datadog-agent.instance-e4e8ed2e-0d7f-11eb-8e5d-a2cf47f4afdc._app.3",
        "short_image:agent-dev"
      ],
      "host": "i-02274f24c2f929bdc",
      "type": "gauge",
      "interval": 0,
      "source_type_name": "System"
    },
```
But let me confirm and review the docs, I believe we copy-pasted this JSON from original docs and then altered.. and then altered some more.

File: deploy/dcos-node-datadog-agent.json (hunk 3 above)
Comment (xornivore) [reply deploy/dcos-node-datadog-agent.json:91]: I'll have to confirm where that came from..

Comment (xornivore) [reply deploy/dcos-node-datadog-agent.json:91]: So the rationale is this http://mesosphere.github.io/marathon/docs/ports.html
But I am not sure if this is correct - in the above section it is targeting the UDP dsd port. @clamoriniere do you know what's up with this (copied from your doc 😄 )

**Pull request merged**